WSEvents.cpp

```diff
@@ -642,6 +653,55 @@ void WSEvents::StreamStatus() {
     obs_output_release(stream_output);
 }
 
+void WSEvents::Heartbeat() {
+    bool streaming_active = obs_frontend_streaming_active();
+    bool recording_active = obs_frontend_recording_active();
+    obs_data_t* data = obs_data_create();
+
+    if (Heartbeat_active) {
+        pulse = !pulse;
+        obs_data_set_bool(data, "pulse", pulse);
+
+        if (Heartbeat_CurrentProfile_active) obs_data_set_string(data, "current-profile", obs_frontend_get_current_profile());
```

Comment:
Okay, I will follow the spec. (personally I like longer lines. 80 came from old monitors.)